cni-plugin/integration/run.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "${BASH_SOURCE[0]%/*}"
+
+# Run kubectl with the correct context.
+function k() {
+  if [ -n "${TEST_CTX:-}" ]; then
+    kubectl --context="$TEST_CTX" "$@"
+  else
+    kubectl "$@"
+  fi
+}
+
+function create_test_lab() {
+    echo '# Creating the test lab...'
+    k create ns cni-plugin-test
+    k create serviceaccount linkerd-cni
+    # TODO(stevej): how can we parameterize this manifest with `version` so we
+    # can enable a testing matrix?
```

Comment:
Something I've done to template out a node-debugger pod manifest whenever I needed to change `nodeName` before applying is to replace the value with a placeholder and then use sed.

```
nodeName: <replace-me>
```

`cat manifest.yaml | sed -e 's/<replace-me>/node-foo/g` | kubectl apply -f -`

I'm not saying it's a good solution but it might do until we move -- if we ever decide to -- to a harness that programatically creates two resources.

Reply:
Now that you mention it, we could use `sed` to replace one image with another image.